packages/components/src/visually-hidden/utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Utility Functions
+ */
+
+/**
+ * renderAsRenderProps is used to wrap a component and convert
+ * the passed property "as" either a string or component, to the
+ * rendered tag if a string, or component.
+ *
+ * See VisuallyHidden hidden for example.
+ *
+ * @param {string|Component} as what tag/component to render
+ * @return {Component} rendered component
+ */
+function renderAsRenderProps( { as: T = 'div', ...props } ) {
```

Comment:
Can we avoid using shortcuts, please. It makes it hard to follow for those not familiar with the pattern:
```suggestion
function renderAsRenderProps( { as: Component = 'div', ...props } ) {
```